docs/policy/service-changes-2018.md

```diff
@@ -0,0 +1,188 @@
+Service Changes 2018
+====================
+
+As you may have heard, the Open Science Grid (OSG) is undergoing some changes to its fabric of services.
+Some services will be retired, most services will be migrated with little planned effect on sites,
+and some services will be migrated with greater effect on sites.
+This document is intended to guide OSG Site Administrators through these changes.
+The following sections list various OSG services and whether or not site administrator action is required.
+
+!!! note
+    There will be announcements prior to retirement or migration of any OSG service.
+
+The OSG Technology team will be holding [office hours](#office-hours) for any questions or comments about these service 
+changes.
+
+OSG CA
+------
+
+The OSG CA service offers certificate request, renewal, and revocation through the [OIM](#myosg-and-oim) web interface, 
+the OIM REST API, and the `osg-pki-tools`.
+This service will be retired but since the OSG CA will remain in the OSG CA distribution,
+your OSG-issued certificates will be valid until they expire.
+Therefore, to extend the window for transitioning to any new CA service, we have the following recommendations:
+
+!!! info "Action item(s)"
+    1. Request fresh certificates for each publicly facing host and service at your site, such as:
+
+        * HTCondor-CE
+        * GlideinWMS VO Frontend: frontend, proxy, and VOMS certificates
+        * GlideinWMS Factory
+        * GridFTP
+        * XRootD
+        * RSV
+        * GUMS
+        * BeStMan
+        * VOMS Admin Server
+
+    1. If you are able to, renew your [user certificate](https://oim.opensciencegrid.org/oim/certificateuser)
+
+In the future, we will use the following CA certificate services:
```

Comment:
In the bullet list, I suggest starting each item with the purpose, then the implementation. OR make a table.